src/main.py

```diff
@@ -65,7 +65,11 @@ def mensa(update, context):
         return
     jsonData = json.loads(resp.content)
     for elem in jsonData:
-        context.bot.send_message(chat_id=update.message.chat_id, text=elem["name"])
+        mealNotes = elem["notes"]
+        if any("vegetarisch" in i for i in mealNotes) or any("vegan" in i for i in mealNotes):
```

Comment:
Couldn't you just use

```python
if "vegetarisch" in mealNotes or "vegan" in mealNotes:
```

?

Reply:
Changed this with the latest commit [](https://github.com/fried-ice/tgMensaBotDD/pull/10/commits/b1a349d10a8bb9b4bf42230fe1a694ca792a4fef)